src/services/snapshots.actions.ts

```diff
@@ -11,13 +11,22 @@ import { Store } from 'src/services/storage'
 import { SetupPage } from 'src/services/setup-page'
 import { Notifications } from 'src/services/notifications'
 import * as IPC from './ipc'
-import { RemovingSnapshotResult, SnapStoreMode } from 'src/types/snapshots'
+import {
+  RemovingSnapshotResult,
+  SnapPanelState,
+  SnapStoreMode,
+  SnapTabState,
+  SnapWindowState,
+  SnapshotState,
+} from 'src/types/snapshots'
 import { Containers } from './containers'
 import { DEFAULT_CONTAINER_ID } from 'src/defaults/containers'
 import { PanelType } from 'src/types/sidebar'
 import { ItemInfo } from 'src/types/tabs'
 import { Info } from './info'
 import { createDefaultSidebarConfig, getSidebarConfigFromV4 } from './sidebar-config'
+import { Favicons } from './favicons'
+import { nextTick } from 'vue'
```

Comment:
Remove `import { nextTick } from 'vue'` (b/c it's unused, and I'm trying to import view lib only in components and top-lvl scripts like sidebar.ts or setup.ts)